apps/angular/src/app/features/anime/anime-details-page/anime-details-page.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="anime">
+	<ng-container *ngIf="anime$ | async as anime">
+		<h2 class="anime__title">{{ anime.titleEnglish | empty }}</h2>
+		<h2 class="anime__title">{{ anime.titleJapanese | empty }}</h2>
+		<div class="anime__details">
+			<img class="details__image" [alt]="anime.titleEnglish" [src]="anime.image" (click)="openDialog(anime.image)" />
```

Comment:
If there are more than 3 attributes, they should be placed in different lines
```suggestion
			<img 
						class="details__image" 
						[alt]="anime.titleEnglish" 
						[src]="anime.image" 
		        			(click)="openDialog(anime.image)" 
			/>
```